test/e2e_node/log_path_test.go

```diff
@@ -39,6 +39,25 @@ var _ = framework.KubeDescribe("ContainerLogPath", func() {
 	f := framework.NewDefaultFramework("kubelet-container-log-path")
 	Describe("Pod with a container", func() {
 		Context("printed log to stdout", func() {
+			BeforeEach(func() {
+				if framework.TestContext.ContainerRuntime == "docker" {
+					// Container Log Path support requires JSON logging driver.
+					// It does not work when Docker daemon is logging to journald.
+					d, err := getDockerLoggingDriver()
+					framework.ExpectNoError(err)
+					if d != "json-file" {
+						framework.Skipf("Skipping because Docker daemon is using a logging driver other than \"json-file\": %s", d)
+					}
+					// Even if JSON logging is in use, this test fails if SELinux support
+					// is enabled, since the isolation provided by SELinux prevents the
+					// checker-container from accessing files owned by the logger-container.
```

Comment:
This isn't true based on your explanation. Can you explain why we skip this test for SELinux a little bit here? Thanks